src/applications/mhv-secure-messaging/components/ThreadList/ThreadsList.jsx

```diff
@@ -89,9 +89,9 @@ const ThreadsList = props => {
         )}
 
         {threadList?.length > 0 && (
-          <div className="xsmall-screen:vads-l-grid-container xsmall-screen:vads-u-max-width--10">
-            <div className="xsmall-screen:vads-l-row xsmall-screen:vads-u-max-width--10">
-              <div className="xsmall-screen:vads-u-margin-left--neg2 xsmall-screen:vads-l-col--12 xsmall-screen:vads-u-padding-left--0 xsmall-screen:vads-u-max-width--10">
+          <div className="xsmall-screen:vads-l-grid-container mobile:vads-u-max-width--100">
+            <div className="xsmall-screen:vads-l-row mobile:vads-u-max-width--100">
+              <div className="xsmall-screen:vads-u-margin-left--neg2 xsmall-screen:vads-l-col--12 xsmall-screen:vads-u-padding-left--0 mobile:vads-u-max-width--100">
```

Comment:
I think it was a typo to use `--10` for these max width classes. That utility name doesn't exist for max width but `--100` does:  https://design.va.gov/foundation/utilities/height-and-width#max-width